src/canvas-extractor/canvas_extractor/mapping/users.py

```diff
@@ -0,0 +1,75 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+import pandas as pd
+
+from . import constants
+
+
+def map_to_udm(users_df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
Not a strong opinion, but you could rename to something like "map_to_udm_users" and then not have to rename when importing multiple of them.

Reply:
✔️